tests/integration_test.rs

```diff
@@ -677,17 +686,23 @@ async fn test_detect_certificate_rotation() {
     let certs_dir = tempfile::tempdir().unwrap();
     let cert_file = certs_dir.path().join("policy-server.pem");
     let key_file = certs_dir.path().join("policy-server-key.pem");
+    let client_cert_file = certs_dir.path().join("client_cert.pem");
```

Comment:
I understand this is not a CA, but we are using it as a client CA in the mTLS configuration. I believe this is fine within the scope of this test. However, could we rename the variable and the file name to `client-ca(.pem)`?